products/products.css

```diff
@@ -0,0 +1,79 @@
+@import "/css/nav.css";
+
+.prd-list {
+  grid-area: main;
+  display: grid;
+  grid-template-columns: auto 1fr;
+  grid-template-rows: 1fr;
+  min-width: 100%;
+  padding: 2rem 0;
+  height: 100%;
+}
+
+.filter-container {
+  display: flex;
+  flex-direction: column;
+  font-size: 1.4rem;
+  margin-left: 2rem;
+  padding: 0 2rem;
+  width: fit-content;
+  border-right: 1px solid var(--nav-text-color);
+  overflow-y: hidden;
+  box-shadow: 4px 8px var(--nav-secondary-color);
+}
+
+.filter-title {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  font-size: 1.5rem;
+  font-weight: 400;
+  padding-bottom: 2rem;
+}
+
+.clr-btn {
+  background: none;
+  border: none;
+  color: var(--icon-count-bg);
+  cursor: pointer;
+  outline: none;
+  font-weight: 500;
+}
```

Comment:
Maybe you can use proper naming here. But its a suggestion tough if you think its correct you can keep it.

Reply:
done